xcell/mappers/mapper_ACT_base.py

```diff
@@ -18,6 +22,18 @@ def _get_ACT_defaults(self, config):
         self.nl_coupled = None
 
     def get_signal_map(self):
+        """
+        Returns the signal map of the children \
+        mapppers of the base class. \
+        If children mappers have already calculated \
+        the signal mapper, the function loads it \
+        from a file. \
+        Otherwise, each mapper's "_get_signal_map()" \
```

Comment:
I don't think this info should be given to the user. If you think it is not clear, add a comment on the source code.